content/providers/01-ai-sdk-providers/25-cohere.mdx

```diff
@@ -106,3 +106,52 @@ Cohere language models can also be used in the `streamText` function (see [AI SD
   The table above lists popular models. You can also pass any available provider
   model ID as a string if needed.
 </Note>
+
+## Embedding Models
+
+You can create models that call the [Cohere embed API](https://docs.cohere.com/reference/embed)
+using the `.embedding()` factory method.
+
+```ts
+const model = cohere.embedding('embed-english-v3.0');
+```
+
+Cohere embedding models support aditional settings. You can pass them as an options argument:
+
+```ts
+const model = cohere.embedding('embed-english-v3.0', {
+  inputType: 'search_document',
+});
+```
+
+The following optional settings are available for Cohere embedding models:
+
+- **inputType** _'search_document' | 'search_query' | 'classification' | 'clustering'_
+
+  Specifies the type of input passed to the model. Default is `search_query`.
+
+  - `search_document`: Used for embeddings stored in a vector database for search use-cases.
+  - `search_query`: Used for embeddings of search queries run against a vector DB to find relevant documents.
+  - `classification`: Used for embeddings passed through a text classifier.
+  - `clustering`: Used for the embeddings run through a clustering algorithm.
```

Comment:
smol nit - all the previous points say "for embeddings" rather than "for the embeddings"